config/common/common.views.xml

```diff
@@ -2598,34 +1910,32 @@
 					<cell type="label" labelfor="name"/>
 					<cell type="field" id="name" name="name" uitype="text"/>
 					<cell type="label" labelfor="cogType"/>
-					<cell type="field" id="cogType" name="cogType" uitype="combobox" picklist="COGTypes"/>
-					<cell type="label" labelfor="parentCojo"/>
-					<cell type="field" id="parentCojo" name="parentCojo" uitype="querycbx" isrequired="false"/>
+					<cell type="field" id="cogType" name="cogType" uitype="combobox"/>
+					<cell type="label" label="Parent COG"/>
+					<cell type="field" id="parentCojo" name="parentCojo" uitype="querycbx"/>
```

Comment:
`parentCojo` was removed in #5406, so this field would have to be changed to `parentCog`

Reply:
Fixed in https://github.com/specify/specify7/pull/5367/commits/5b150d5cdc3be2a2aeb3c38d43b3b839f5bbf4cd